ros/build.sh

```diff
@@ -0,0 +1,4 @@
+cd ~/CarND-CapStone/ros
```

Comment:
I have it in a different directory. Others may also keep it in different directories.

Reply:
To execute the script we need to navigate to ros folder anyway, so it's better if we remove this line and keep the rest.